### DISCUSSION TITLE
Archivos creados en carpeta de trabajo

### DISCUSSION THREAD

**Discussion by Mikson16** [IIC2133-PUC/2024-2#52] [Tarea 0] [answered]
Hola, tengo una duda, a medida que he ido desarrollando mi trabajo me han aparecido un par de archivos en mi carpeta de trabajo, del cual desconozco su procedencia y no estoy seguro si agregarlos al repositorio  o bien excluirlos en él .gitignore.

Adjunto una imagen de los archivos.
![image](https://github.com/user-attachments/assets/70bd48a4-6274-42a3-8459-f4530d62cb88)

**Comment by 4n1t4** [comment, marked as answer]
hola!  Si te das cuentas por default el gitignore ignora todos los archivos dentro de `.vscode`. ademas de los ejecutables que se crean una vez compilas. Mientras no modifiques el gitignore deberia estar okey

**Reply by Mikson16** [reply]
A decir verdad, para evitar que se cuenten los archivos dentro de .vscode, tuve que agregarlo manualmente al .gitignore. Pero muchas gracias, ya con eso me quedo más tranquilo.
